frontend/src/components/NavBar.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import "./style/NavBar.scss";
+
+class NavBar extends Component {
```

Comment:
Seems to me this one could have been a function, don't you think ? 
(That's not a blocker, just a question)